City-of-Helsinki/drupal-hdbt
# UHF-2286: Add styles that override kuura-widget-container-inline styl…

…es, make attachments region content visible for users

How to test (existing install)

1. On project root run `composer require drupal/hdbt:dev-UHF-2286-kuura-chat-positioning && composer require drupal/helfi_platform_config:dev-UHF-2286-kuura-chat-positioning`
2. `make drush-cr`
3. If you don't have Kuura Health Chat visible yet go to `/admin/structure/block` and scroll down. There should be a region called Attachments. Add a block there called Kuura Health Chat. Save the configuration page.
4. Go to your local site using incognito window or remove all your cookies and you should see the cookie-popup. The Kuura Health Chat should no longer be above it. Accept cookies and you should be able to see the Kuura Health Chat once the popup has been lifted.

Also check this PR: https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/152
